libs/input-mapping/tests/Functional/DownloadFilesTest.php

```diff
@@ -274,7 +274,7 @@ public function testReadFilesLimit(): void
                 (new FileUploadOptions())->setTags([$this->testFileTag]),
             );
         }
-        sleep(5);
+        sleep(10);
```

Comment:
Vytvari se tu 100+ souboru a tech 5s je zda se nedostatecnych. Obcas to pada i normalne.

Reply:
to je hrozně moc, nemůžem tam dát settle?

Reply:
Jo, upravím to. Ten by tomu vlastně měl pomoc.

Reply:
Narvu ho i do OM, tam to neni kriticky, ale at ty sleepy pak lidi nematou.

Reply:
Díval jsem se na to předtím z kontextu toho že jsem ty testy zkrátil na polovinu, tak pár sekund nikoho nerozhází :D

Reply:
jj, to jo, ale snažil bych se jich zbavit jestli to není mega voser

Reply:
Tak ten settle je dost na pytel.

Zkousel jsem to lokalne. Mam to udelany tak, ze ceka az je dostupny file, ktery jsem zalozil naposled. Upravil jsem vetsinu testu, kde je sleep po zalozeni souboru. No a kdyz se tam zaklada vice souboru, tak se muze stat ze ten posledni je v indexu dostupnej driv, nez ty predchozi a testy failnou.

Takze sleep je v tuhle chvili nejlepsi reseni. Do budoucna to pocitam zmizi, budou files dostupny jen z mysql tabulky a ne elasticu.

Reply:
Nakonec jsem to predelal na ten settle. Tady v tomhle konkretnim pripade vim, kolik souboru zakladam. Takze je to v pohode. Tech 10s nestacilo